### ISSUE TITLE
Remove site path from automatic instrumented resource name

### ISSUE THREAD

**Issue by geofflamrock** [DataDog/dd-trace-dotnet#594]
Hi,

Our application is installed into IIS with a specific virtual path in order to be able to support multiple applications on the same machine for different instances e.g. "/api-instance1" and "/api-instance2". 

When automatic instrumentation creates the resource name it includes this path e.g. "GET /api-instance1/api/users" which makes it difficult to search and compare traces etc. across different setups as the resource name is always unique.

Is there a way to configure the automatic instrumentation to exclude the virtual path component so that the resource name is consistent?

Please let me know if anything is unclear with what I've described or if you need any more information.

Thanks.

**Comment by lucaspimentel** [comment]
@geofflamrock: Thanks for the feedback. There is no way to configure this currently, but it sounds like a good feature to add for all the web integrations. To be clear, you would like paths in resource names to look something like this?
```
(proposed)
GET ~/api/users
```
or simply
```
(proposed)
GET api/users
```
instead of what you see now
```
(current)
GET /api-instance1/api/users
```
when `/api-instance1` is the virtual path in IIS, correct?

**Comment by geofflamrock** [comment]
@lucaspimentel yes it would be good to see

```
GET api/users
```

instead of

```
GET /api-instance1/api/users
```

Interestingly, looking at the outputted spans in DataDog, there are 2 spans for every trace, one with the virtual path and then a nested one without it.

![image](https://user-images.githubusercontent.com/2915931/71126798-d9c2e780-223d-11ea-862a-50c046fae903.png)

**Comment by lucaspimentel** [comment]
This issue was resolved in #1288 with the addition of feature flag `DD_TRACE_ASPNET_ROUTE_TEMPLATE_RESOURCE_NAMES_ENABLED`, which will become the default soon.
